Makefile

```diff
@@ -1,10 +1,10 @@
 # Image URL to use all building/pushing image targets
-IMG ?= europe-docker.pkg.dev/kyma-project/prod/telemetry-manager:v20230421-c40cd7f7
+IMG ?= europe-docker.pkg.dev/kyma-project/prod/telemetry-manager:v20230505-50348605
```

Comment:
Also set this version in the kustomization to have it applied to the module

Reply:
The kustomization file will be automatically be changed via the make target https://github.com/kyma-project/telemetry-manager/blob/main/Makefile#L186